internal/querynodev2/segments/segment_loader.go

```diff
@@ -890,6 +890,17 @@ func (loader *segmentLoader) loadSealedSegment(ctx context.Context, loadInfo *qu
 				return err
 			}
 		}
+
+		if !storagecommon.IsVectorDataType(field.GetDataType()) &&
```

Comment:
Is this code snippet related to drop index?

Reply:
> Is this code snippet related to drop index?

yes, we load all raw data without warmup to make it searchable after dropping index.